### DISCUSSION TITLE
Would be great to see 7 days, not only 4 on the main screen

### DISCUSSION THREAD

**Discussion by NadinSh** [iSoron/uhabits#1296] [Feature Requests]
Hello, the issue is actually described already. We can see only 4 days on the main screen which doesn't seem to be sufficient and motivating. If it's possible to change, please help since I've failed to find this option in the settings.

UPD in the reply below, I've added a screenshot with my habit tracker

**Comment by NadinSh** [comment]
![Screenshot_2022-02-20-11-22-42-79_7a4090f09f6554852d748ee9fd6f40d3](https://user-images.githubusercontent.com/100068166/154836232-85a5bd0a-c839-4bc1-bc09-0d50d17a0c28.jpg)

**Reply by NadinSh** [reply]
I've added a screenshot where it's shown that even if I left only tiny icons, still don't see the whole week

**Reply by selimyo** [reply]
That's a fair point about the blank space not being used (when you choose to put icons for example). 

In the meantime, if you want to see more of the past days I suggest either rotating your phone to horizontal position (shows up to 10 days total on my phone), or holding down the top of the screen to scroll right-->left through the previous days. Sorry if you're already aware of these features (and I know they're not what you're looking for exactly).

**Reply by baincd** [reply]
@selimyo,

> ...holding down the top of the screen to scroll right-->left through the previous days.

Whoa!  I was not aware this was a feature.  That's awesome, thank you for pointing that out! :smiley:
